src/main/kotlin/org/amshove/kluent/Basic.kt

```diff
@@ -1,31 +1,32 @@
 package org.amshove.kluent
 
-import org.junit.Assert.*
 import kotlin.reflect.KClass
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
 
-infix fun Any?.shouldEqual(theOther: Any?) = assertEquals(theOther, this)
+infix fun Any?.shouldEqual(theOther: Any?) = assertEquals(this, theOther)
```

Comment:
Parameter positions matter, first is `expected`, second is `actual`, so the correct invocation is `assertEquals(theOther, this)`